### PR TITLE
Customise feed titles

### PR DESCRIPTION
**Pull request by shaneHowearth**
Added a field to the database that is populated with the display name for the feed.

The config file feeds strings will be split on the '~' char, with the first section being the url to fetch, and the second (optional) section being the name displayed in the window.

This *only* changes the name in the feeds window, *not* the name in the article window - please let me know if a change is preferred

This change will not work with an existing copy of the database, to use this version will require execution of the following command (assuming this is where the database is on your local system) `mv ~/.local/share/gorss/gorss.db{,.bkup}`

Currently the display is tightly coupled to the fetching of the URL, I feel that this should be re-architected, but this PR may not be the right place to do that.

### CODE REVIEW

**Comment by Lallassu** [comment]
I tried it out and it looks good! ping me when it's ready to merge.

**Comment by shaneHowearth** [comment]
@Lallassu I've tidied up the PR a little, split it into several commits, to try and make the change clearer(although the second and third commit have a bit of a mix... but I don't want to risk losing stuff)

If you would like mt to do any more to this PR feel free to let me know

**Comment by shaneHowearth** [comment]
Hrm, this is going to break things for clients that have existing feeds that have ~ in the URL ( as seen in the issue discussion) - I want to put a hold on this from being merged until a more robust solution is worked out.

The natural inclination for me is to write a space then a ~between the URL and the optional name, which is what I think I will change the delimiter to.

**Comment by shaneHowearth** [comment]
@Lallassu Assuming this passes your tests, I'm happy for it to be merged now

**Comment by shaneHowearth** [comment]
Hrm, this will break peoples' existing database, should I write something to catch that (can a sqllite table be altered)? Or is it better to release this with a note on the database needing a `mv ~/.local/share/gorss/gorss.db{,-old}`

**Comment by Lallassu** [comment]
> Hrm, this will break peoples' existing database, should I write something to catch that (can a sqllite table be altered)? Or is it better to release this with a note on the database needing a `mv ~/.local/share/gorss/gorss.db{,-old}`

I think a `mv` will do and include this in a major release later on. Like a 1.0.0. I've not been that strict to versions yet, but I'll start with correct versioning and that will basically solve any breaking change.

File: README.md [hunk 1]
```diff
@@ -71,10 +75,10 @@ It's possible to specify configuration file as a flag, default is `gorss.conf`.
     "OPMLFile": "../example_ompl.xml",
     "feeds": [
         "https://news.ycombinator.com/rss",
-        "https://www.sweclockers.com/feeds/nyheter",
-        "https://www.reddit.com/r/homeassistant/.rss",
-        "https://www.reddit.com/r/golang/.rss",
-        "https://www.reddit.com/r/programming/.rss"
+        ("url": "https://www.sweclockers.com/feeds/nyheter", "name": "Swedish Overclocking"},
+        ("url": "https://www.reddit.com/r/homeassistant/.rss", "name": "Home Assistant"},
+        ("url": "https://www.reddit.com/r/golang/.rss"},
```
Comment (vaygr) [inline README.md:80]: I think there's a typo here on these 3 lines with `(` instead of `{`.

Comment (Lallassu) [reply README.md:80]: Good catch. I fixed that directly in master branch.

**Pull request merged**